digits/model/images/generic/test_views.py

```diff
@@ -595,9 +601,15 @@ def test_infer_many_from_folder(self):
         # StringIO wrapping is needed to simulate POST file upload.
         file_upload = (StringIO(textfile_images), 'images.txt')
 
+        # try selecting the extension explicitly
+        extension = extensions.view.get_default_extension()
+        extension_id = extension.get_id()
+
         rv = self.app.post(
                 '/models/images/generic/infer_many?job_id=%s' % self.model_id,
-                data = {'image_list': file_upload, 'image_folder': os.path.dirname(self.test_image)}
+                data = {'image_list': file_upload,
+                        'image_folder': os.path.dirname(self.test_image),
+                        'view_extension_id': extension_id}
```

Comment:
If you don't specify a view extension then the default one (raw data) is used (see [there](https://github.com/gheinrich/DIGITS/blob/747f462ffe08fd63d6a62375c50cca2ac70166e0/digits/extensions/view/__init__.py#L12) ).